GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -70,6 +80,11 @@ public void InitializeDefaultParameterValues()
 
         public void Execute()
         {
+            if (this.StartedByVerb)
+            {
+                GVFSPlatform.Instance.PrepareProcessToRunInBackground();
```

Comment:
Can you elaborate on why this is only needed when we call from the MountVerb?  Some comments would be great :-)